File: code_snippets/python-sdk/workflows/text_classifier.py
from clarifai.client.workflow import Workflow

# Your PAT (Personal Access Token) can be found in the Account's Security section

# Initialize the workflow_url
workflow_url = "https://clarifai.com/clarifai/main/workflows/Text-Moderation"
text_classfication_workflow = Workflow(
    url= workflow_url , pat="YOUR_PAT"
)
raw_text = b"I love your product very much"
result = text_classfication_workflow.predict_by_bytes(raw_text, input_type="text")
print(result.results[0].outputs[0].data)

File: code_snippets/python-sdk/workflows/textClassifier.ts
import { Workflow } from "clarifai-nodejs";


const workflowUrl =
  "https://clarifai.com/clarifai/main/workflows/Text-Moderation";

const textClassificationWorkflow = new Workflow({
  url: workflowUrl,
  authConfig: {
    pat: process.env.CLARIFAI_PAT,
  },
});

const rawText = Buffer.from("I love your product very much");
const prediction = await textClassificationWorkflow.predictByBytes(
  rawText,
  "text",
);
console.log(prediction.resultsList?.[0]?.outputsList?.[0]?.data);
